js/index.js

```diff
@@ -0,0 +1,72 @@
+class Card {
```

Comment:
It would be nice if you store all Card objects in a state variable, like an array: let cards = []. That would help to keep track of these objects and make adjustments in the future. For example, if it is asked to add a button to shuffle cards, in that case, you can easily shuffle the array and render new cards accordingly